pyproject.toml

```diff
@@ -9,7 +9,7 @@ authors = [
   { name = "Gregor Lenz", email = "gregor@neurobus.space" },
 ]
 requires-python = ">=3.9"
-dependencies = ["numpy>=1.26"]
+dependencies = ["numpy>=2"]
```

Comment:
Are you using features of numpy that require 2.x? Otherwise, I reckon that we should try and preserve compatibility with numpy 1 (2.x will still be installed by default but users can choose 1.x if they are using Faery alongside a library that is not compatible with 2.x).

Reply:
That's an interesting point. I think my instinct just wanted to use the same version for Rust as for Python, but you're right that that doesn't have to be the case at all. I'll revert this.